src/platform/silabs/wifi/wf200/WifiInterface.cpp

```diff
@@ -293,6 +293,18 @@ sl_status_t get_all_counters(void)
     return result;
 }
 
+/**
+ * @brief Convert RCPI to RSSI
```

Comment:
Missing comments when the result of the formula doesn't fit in a int16

Reply:
Added comments